base/applications/games/solitaire/lang/sq-AL.rc

```diff
@@ -56,7 +56,7 @@ END
 STRINGTABLE
 BEGIN
     IDS_SOL_NAME "Solitaire"
-    IDS_SOL_ABOUT "Solitaire nga J Brown\n\nCardLib version 1.0."
+    IDS_SOL_ABOUT "Solitaire nga J Brown\nCardLib version 1.0."
```

Comment:
Some languages do end this line with  dot after the 1.0 and some like sk-SK.rc don't do that. I do recommend for the purpose of harmonization to strip it everywhere.

Reply:
Done.